packages/orbit-components/src/primitives/ButtonPrimitive/__tests__/index.test.tsx

```diff
@@ -87,16 +87,15 @@ describe("ButtonPrimitive", () => {
 
   it("should behave as button", async () => {
     const children = "Lorem ipsum";
-    const dataTest = "test";
     const onClick = jest.fn();
 
     render(
-      <ButtonPrimitive dataTest={dataTest} onClick={onClick} tabIndex={0} asComponent="div">
+      <ButtonPrimitive onClick={onClick} tabIndex={0} asComponent="div">
```

Comment:
I thought about that but I removed it because we already do `getByTestId` (and therefore we already test the `dataTest` prop) on the first test, so I don't think we need it here 🤔